doc/superbuild/source/suncontrol/index.rst

```diff
@@ -0,0 +1,20 @@
+..
+   Daniel R. Reynolds @ SMU
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2023, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. include:: ../../../shared/suncontrol/SUNControl_Description.rst
```

Comment:
I don't quite follow -- there is indeed a `SUNAdaptController_links.rst` file in this folder (mirroring the structure of all other subdirectories inside `superbuild/source`).  Perhaps if you just make the change that you're recommending it would be faster all the way around?

Reply:
I've moved this line inside `SUNAdaptController_links.rst`. Other subdirectories have a brief introduction to the chapter which appears before the first section by adding the `include` before the `toctree`. In this case the material that would normally be the first section was appearing as the intro to the chapter.